lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -1566,3 +1755,21 @@ def test_pagination(self):
             404,
             {"developer_message": "Page not found (No results on this page)."}
         )
+
+    def test_author_profile_image(self):
+        self.register_get_user_response(self.user)
+        cs_comment_child = self.make_comment_data("test_child_comment", self.comment_id, children=[])
+        cs_comment = self.make_comment_data(self.comment_id, None, [cs_comment_child])
+        cs_thread = make_minimal_cs_thread({
+            "id": self.thread_id,
+            "course_id": unicode(self.course.id),
+            "children": [cs_comment],
+        })
+        self.register_get_thread_response(cs_thread)
+        self.register_get_comment_response(cs_comment)
+        self.create_profile_image()
```

Comment:
`self.create_profile_image ` does that implicitly; it creates and checks the successful creation of image. Please refer to its docstring